packages/codemirror-extension/src/services.tsx

```diff
@@ -62,17 +63,22 @@ export const languagePlugin: JupyterFrontEndPlugin<IEditorLanguageRegistry> = {
       name: 'ipythongfm',
       mime: 'text/x-ipythongfm',
       load: async () => {
-        const [m, tex] = await Promise.all([
+        const [m, py, tex] = await Promise.all([
           import('@codemirror/lang-markdown'),
+          import('@codemirror/lang-python'),
           import('@codemirror/legacy-modes/mode/stex')
         ]);
-        return m.markdown({
+        const mdlang = m.markdown({
           base: m.markdownLanguage,
           codeLanguages: (info: string) => languages.findBest(info) as any,
           extensions: [
             parseMathIPython(StreamLanguage.define(tex.stexMath).parser)
           ]
         });
+        return new LanguageSupport(
+          mdlang.language,
+          pythonBuiltin(py.pythonLanguage)
```

Comment:
Nice, you are only missing to add the default support from markdown:

```suggestion
          [mdlang.support, pythonBuiltin(py.pythonLanguage)]
```

And this is my final comment for this PR.